README.md

```diff
@@ -128,6 +151,65 @@ client2
      - wireguard
 ```
 
+### If wireguard_manage_keys is `True`
+All hosts is servers and clients (peer2peer).
+
+```yaml
+# group_vars/all.yml
+wireguard_listen_port: 5888
+wireguard_wg0_preshared_key: secret_preshared_key
+wireguard_wg0_peer_settings: >
+  {%  set _peers = {} -%}
```

Comment:
I didn't know one could do jinja templating within host/group vars files. Could you point me to docs where this is described?

Reply:
I did not find a direct indication in the official documentation. But here is a link to changing variables using the filters https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#transforming-variables-with-jinja2-filters
and a links to the possibility of using a jean in the values ​​of variables:
https://groups.google.com/forum/m/#!topic/ansible-project/ZDz-8tcsdTA

Probably, I can transfer this part of the code to the template, but this can lead to loss of flexibility in the settings. In any case, I'll think about how to improve the code. If you have any suggestions, I will be glad to hear them.
P.S. Sorry for the delay in response, I was on vacation.
P.P.S. This PR is an attempt to combine the functionality of your role and this role https://github.com/mawalu/wireguard-private-networking